apps/web/app/(use-page-wrapper)/settings/(settings-layout)/organizations/teams/other/page.tsx

```diff
@@ -1,25 +0,0 @@
-import { _generateMetadata, getTranslate } from "app/_utils";
-
-import LegacyPage from "@calcom/features/ee/organizations/pages/settings/other-team-listing-view";
-import SettingsHeader from "@calcom/features/settings/appDir/SettingsHeader";
-
-export const generateMetadata = async () =>
-  await _generateMetadata(
-    (t) => t("org_admin_other_teams"),
-    (t) => t("org_admin_other_teams_description"),
-    undefined,
-    undefined,
-    "/settings/organizations/teams/other"
-  );
-
-const Page = async () => {
-  const t = await getTranslate();
-
-  return (
```

Comment:
moving to (main-page) route group because otherwise `loading.tsx` gets applied to all nested routes in `/settings/organizations/teams`